tests/html_vue/jsfmt.spec.js

```diff
@@ -1,3 +1,5 @@
 run_spec(__dirname, ["vue"]);
 run_spec(__dirname, ["vue"], { trailingComma: "es5" });
 run_spec(__dirname, ["vue"], { semi: false });
+run_spec(__dirname, ["vue"], { vueIndentScriptAndStyle: true });
+run_spec(__dirname, ["vue"], { vueIndentScriptAndStyle: false });
```

Comment:
Could you make a new folder `tests/html_vue_indent/` and put the new stuff in there? It feels like most of the existing tests didn’t benefit from being run with `vueIndentScriptAndStyle true/false` and only generated a huge diff.

Also, could you add a test case with `<script>` and `<style>` tags _inside_ `<template>`? Just in case. Those should follow our HTML rules and always be indented.

Reply:
Sure.